packages/terra-clinical-header/src/Header.scss

```diff
@@ -26,6 +26,7 @@
     align-self: center;
     flex: 0 0 auto;
     height: 100%;
+    margin: 0 10px;
```

Comment:
Adding the left and right margins will need sign off from UX. I'd reach out to @neilpfeiffer and see if this is how we want the solve the spacing issue with the Header (Start content and Title having no separation)

It might be worth fixing the alignment in Safari without adding the margins on this pull request if there is additional discussion needed for the spacing

Reply:
Yeah, I made a mistake and addressed the margins instead of vertical alignment - I'll revert this commit then push up a fix for the alignment.